organization/views.py

```diff
@@ -184,21 +184,39 @@ def get_reference_organization(self):
         return Organization.objects.get(uuid=self.kwargs['organization_id'])
 
     def get_success_url(self):
-        uuid = self.kwargs[self.slug_field_kwarg]
-        return reverse('view-organization', kwargs={'slug': uuid})
+        organization_id = self.kwargs['organization_id']
+
+        # when saving and continuing, always return to the page we were on
+        if self.request.POST.get('_continue'):
+            return self.request.path
+        else:
+            return reverse('view-organization', kwargs={'slug': organization_id})
 
     def get_cancel_url(self):
         return reverse('view-organization', kwargs={'slug': self.kwargs['organization_id']})
 
 
 class OrganizationCreateView(BaseCreateView):
 
+    def get_success_url(self):
+        organization_id = self.kwargs['organization_id']
+
+        # when saving and continuing, always return to the page we were on
+        if self.request.POST.get('_continue'):
+            return self.request.path
```

Comment:
This Save & Continue redirect doesn't quite work as expected for CreateViews, since `self.request.path` will take the user back to the CreateView instead of to the edit view for the freshly-created entity. The UUID of the new entity is available on the form object returned by `self.get_form()`, we just need to figure out what the corresponding edit URLpattern is.

I can think of a couple ways that we could address this:

1. Define a method (like `get_save_and_continue_url`) on all children of `OrganizationCreateView` that returns the proper URL for Save & Continue
2. Define a class attribute (like `edit_url_name`) on all children of `OrganizationCreateView` that records the corresponding edit path (e.g. `OrganizationCreateBasicsView.edit_url_name` would be `'edit-organization'`) and then reference that attribute in this method along with `self.get_form()` to construct the proper URL

I think choice 2) is more DRY, but comes at the cost of introducing yet another class attribute in this module. Curious to hear what you think!